apps/dolly-frontend/src/main/js/src/components/bestillingsveileder/stegVelger/steg/steg1/paneler/Familierelasjoner.tsx

```diff
@@ -76,16 +89,30 @@ export const FamilierelasjonPanel = ({ stateModifier, formValues }) => {
 						''
 					}
 				/>
-				<Attributt
-					attr={sm.attrs.foreldreansvar}
-					disabled={npidPerson || (leggTilPaaGruppe && harTestnorgeIdenter)}
-					title={
-						(npidPerson && 'Ikke tilgjengelig for personer med identtype NPID') ||
-						(leggTilPaaGruppe && harTestnorgeIdenter && tekstLeggTilPaaGruppe) ||
-						''
-					}
-					vis={!testNorgePerson}
-				/>
+				{kanHaForeldreansvar && (
```

Comment:
Istedenfor å ha to sjekker her så kan vel dette byttes med bare å ha sjekken i selve attr prop:
> <Attributt
						attr={kanHaForeldreansvar ? sm.attrs.foreldreansvar : sm.attrs.ansvarssubjekt}
						disabled={npidPerson || (leggTilPaaGruppe && harTestnorgeIdenter)}
						title={
							(npidPerson && 'Ikke tilgjengelig for personer med identtype NPID') ||
							(leggTilPaaGruppe && harTestnorgeIdenter && tekstLeggTilPaaGruppe) ||
							''
						}
						vis={!testNorgePerson}
					/>
					
siden disse to komponentene er helt like unntatt det som er i attr prop og kanHaForeldreansvar sjekken her alltid vil gi kun en komponent tilbake

Reply:
Super tilbakemelding, tusen takk! 😃